tee-worker/ts-tests/indirect_error_calls.ts

```diff
@@ -149,3 +150,79 @@ export async function removeErrorIdentities(
     }
     return undefined;
 }
+export async function requestErrorVCs(
+    context: IntegrationTestContext,
+    signer: KeyringPair,
+    aesKey: HexString,
+    listening: boolean,
+    mrEnclave: HexString,
+    assertion: Assertion
+): Promise<
+    | {
+          account: HexString;
+          index: HexString;
+          vc: HexString;
+      }[]
+    | undefined
+> {
+    let txs: TransactionSubmit[] = [];
+    let len = 0;
+
+    for (const key in assertion) {
+        len++;
+        const tx = context.substrate.tx.vcManagement.requestVc(mrEnclave, {
+            [key]: assertion[key as keyof Assertion],
+        });
+        const nonce = await context.substrate.rpc.system.accountNextIndex(signer.address);
+
+        let newNonce = nonce.toNumber() + (len - 1);
+        txs.push({ tx, nonce: newNonce });
+    }
+
+    await sendTxUntilInBlockList(context.substrate, txs, signer);
+
+    if (listening) {
+        const events = (await listenEvent(context.substrate, 'vcManagement', ['StfError'])) as any;
+        expect(events.length).to.be.equal(len);
+        return events;
+    }
+    return undefined;
+}
+export async function disableErrorVCs(
+    context: IntegrationTestContext,
+    signer: KeyringPair,
+    listening: boolean,
+    indexList: HexString[]
+): Promise<string[] | undefined> {
+    let txs: TransactionSubmit[] = [];
+
+    for (let k = 0; k < indexList.length; k++) {
+        const tx = context.substrate.tx.vcManagement.disableVc(indexList[k]);
+        const nonce = await context.substrate.rpc.system.accountNextIndex(signer.address);
+        let newNonce = nonce.toNumber() + k;
+        txs.push({ tx, nonce: newNonce });
+    }
+
+    const res = (await sendTxUntilInBlockList(context.substrate, txs, signer)) as string[];
+
+    return res.length ? res : undefined;
+}
+export async function revokeErrorVCs(
+    context: IntegrationTestContext,
+    signer: KeyringPair,
+    listening: boolean,
+    indexList: HexString[]
+): Promise<string[] | undefined> {
+    let txs: TransactionSubmit[] = [];
+
+    for (let k = 0; k < indexList.length; k++) {
+        const tx = context.substrate.tx.vcManagement.revokeVc(indexList[k]);
+        const nonce = await context.substrate.rpc.system.accountNextIndex(signer.address);
```

Comment:
A small improvement: I think we can move it out of the loop - we'll get the same result each time anyway.